SEED-platform/seed
# develop - Update 0119 migration with a state_operation to align migration state

#### Any background context you want to provide?
`recognize_empty` was added to the column table as a patch in a previous release, 2.6.1. A modified version of that migration needed to be included in later releases like this so that databases could be carried over easily between release upgrades.

#### What's this PR do?
This PR further modifies that migration to align Django's underlying migration state.

#### How should this be manually tested?
With a DB that already has `recognize_empty` on the column table, run this updated migration.
Then verify that running `./manage.py makemigrations` leads to `No changes detected`.

Ideally, check out `2.6.1-patch3`, create a new DB, run migrations, check out this branch and finishing running all the migrations. Finally verify with `makemigrations` as described above.

#### What are the relevant tickets?
#### Screenshots (if appropriate)
